### ISSUE TITLE
Able to navigate to invisible parts of Creation journeys (Create Plan / Product / Feature) via tab controls

### ISSUE THREAD

**Issue by adaptajoe** [Salable/gds#237]
Using Tab controls, a user is able to navigate to invisible elements further on / previously in the journey; this is due to elements with `aria-hidden='true'` still maintaining their `TabIndex`.

This is exhibited on:

- Create Product journey
- Create Feature journey
- Create Plan journey

**Comment by adaptajoe** [comment]
This should be in the Salable JIRA rather than here. Closing.
